guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<p>Could you also briefly explain what zero-shot classification is. It will be helpful to the user.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>done</p>